<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Keyboard</class>
 <widget class="QWidget" name="Keyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>202</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>287</width>
    <height>202</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>287</width>
    <height>202</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Keyboard</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="one">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="two">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="three">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="four">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="five">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="six">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="seven">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="eight">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nine">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zero">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="q">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>q</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="w">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>w</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="e">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>e</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="r">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>r</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="t">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>t</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="yB">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="u">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>u</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="i">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>i</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="o">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>o</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="p">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>p</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="a">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>a</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="s">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="d">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>d</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="f">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>f</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="g">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>g</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="h">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>h</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="j">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>j</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="k">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>k</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="l">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>l</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="apostrButtton">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>'</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="z">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="xB">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>c</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="v">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>v</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>b</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="n">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>n</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>m</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>&lt;-</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="periodButton">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="spaceButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>space</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="commaButton">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextLineButton">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>\n</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
